src/random_number.py

```diff
@@ -0,0 +1,8 @@
+from random import randint
+
+
+def main():
+    print("The random number is:", randint(1, 100))
+
+
+main()
```

Comment:
Aggiungi un 
```python
if __name__ == "__main__":
    main()
```
come [qui](https://www.geeksforgeeks.org/what-does-the-if-__name__-__main__-do)